themes/custom/az_barrio/config/install/az_barrio.settings.yml

```diff
@@ -79,7 +79,7 @@ info_security_privacy: true
 copyright_notice: 'The Arizona Board of Regents on behalf of <a href="https://www.arizona.edu" target="_blank">The University of Arizona</a>.'
 az_hide_front_title: false
 az_back_to_top: true
```

Comment:
I think we will need a post update function to covert this setting from
 if `az_barrio_navbar_offcanvas: true` then `az_barrio_navbar: true`

Additionally, we'll need to add `az_barrio_navbar` to the az_barrio.schema.yml and remove `az_barrio_navbar_offcanvas`